latency: 0.0
n_objects: 20